src/components/ColorPicker/commons/saturation/index.js

```diff
@@ -42,14 +42,22 @@ const Saturation = React.forwardRef((_props, ref) => {
     };
 
     const unbindEventListeners = () => {
-        window.removeEventListener('mousemove', handleChange);
-        window.removeEventListener('mouseup', unbindEventListeners);
+        containerRef.current.removeEventListener('mousemove', handleChange);
+        containerRef.current.removeEventListener('mouseup', unbindEventListeners);
+        // eslint-disable-next-line no-use-before-define
+        containerRef.current.removeEventListener('mouseleave', handleMouseLeave);
```

Comment:
can we manage all events directly in the element like ```onMouseDown={handleMouseDown}```?
this approach is always preferable over using addEventListener and removeEventListener